docs/r2/graphs.ipynb

```diff
@@ -384,18 +384,108 @@
       },
       "cell_type": "markdown",
       "source": [
-        "<img class=\"tfo-display-only-on-site\" src=\"https://github.com/tensorflow/tensorboard/blob/master/docs/r2/images/graphs_tag_selection.png?raw=1\"/>\n",
+        "<img class=\"tfo-display-only-on-site\" src=\"https://github.com/tensorflow/tensorboard/blob/master/docs/r2/images/graphs_tag_selection.png?raw=1\"/> <br/>\n",
         "<img class=\"tfo-display-only-on-site\" src=\"https://github.com/tensorflow/tensorboard/blob/master/docs/r2/images/graphs_conceptual.png?raw=1\"/>"
       ]
     },
     {
       "metadata": {
-        "id": "yBmh8RPI8CcZ",
+        "id": "zVuaKBifu-qF",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "## Graphs of tf.functions\n",
+        "\n",
+        "The examples so far have described graphs of Keras models, where the graphs have been created by defining Keras layers.\n",
+        "\n",
+        "The [```tf.function```](https://www.tensorflow.org/r2/guide/autograph) annotation in TensorFlow 2.0 allows you to transform a Python computation function into portable, high-performance TensorFlow graphs. This is called **autographing** a function. The **TensorFlow Trace API** generates graph data for autographed functions for visualization in TensorBoard."
```

Comment:
Not sure what reads better:
`TensorFlow Trace API` -> `TensorFlow Summary Trace API`

Reply:
Thanks, fixed